src/models/mod.rs

```diff
@@ -7,10 +7,12 @@ mod enemy;
 mod particle;
 mod player;
 mod world;
+mod star;
 
 pub use self::bullet::Bullet;
 pub use self::enemy::Enemy;
 pub use self::particle::Particle;
 pub use self::player::{Player, POLYGON as PLAYER_POLYGON};
 pub use self::vector::Vector;
 pub use self::world::World;
+pub use self::star::Star;
```

Comment:
A newline is missing here

Reply:
Hmm something strange is happening here - when I checkout the repo locally there's a newline at the end here, as well as when I attempt to edit it online using GitHub's online editor... But when I preview it online, there's no newline?

Before we merge I'll squash all the commits down and maybe it will fix this? Never seen this before.